index.js

```diff
@@ -55,15 +55,66 @@ const sendEmail = async ({
       expiredKeys,
     }),
   }
-  sgMail.send(msg)
+  //sgMail.send(msg)
 }
 
+const getKeyDetails = async ({
+  keyNames
+}) => {
+  const keyInfoFileContents = await readFileAsync(join(__dirname, './keys.info'))
+  var keysToReturn = {}
+  keyListInfo = JSON.parse(keyInfoFileContents)
+  keyListInfo.keys.forEach((keyInfo) => {
+    if (keyNames.indexOf(keyInfo.name) != -1) {
+      keysToReturn[keyInfo.name] = keyInfo
+    }
+  })
+  return keysToReturn
+}
+
+const sendRemindersForExpiredKeys = async({
+  expiredKeys
+}) => {
+  console.log(expiredKeys)
+  expiredKeyMailTemplate = await readFileAsync(join(__dirname, './reminderemail.html'))
+  keyListInfo = await getKeyDetails({
+    keyNames: expiredKeys,
+  })
+  
+  expiredKeys.forEach((expiredKey) => {
+    
+    var compiledExpiredEmail = compile(expiredKeyMailTemplate.toString())
+    if (keyListInfo[expiredKey] && keyListInfo[expiredKey].recipients){
+      console.log("sending a mail for " + expiredKey)
+      var htmlToSend = compiledExpiredEmail({
+        recipients: keyListInfo[expiredKey].recipients.map(recipient => {return recipient.name}).join("/"),
+        keyToRotate: expiredKey
+      })
+      var msg = {
+        to: keyListInfo[expiredKey].recipients.map(recipient => {return recipient.email}),
```

Comment:
js tip: this could omit the `{}` and `return`: `recipients.map(recipient => recipient.email)`